### ISSUE TITLE
 ReferenceError: _ is not defined

### ISSUE THREAD

**Issue by aiveneh** [ashvin27/react-datatable#52]
```
ReferenceError: _ is not defined
    at TableHeader (TableHeader.js:38)
    at renderWithHooks (react-dom.development.js:16260)
```

I got this when trying to render a table, looks like the proper lodash package wasn't included in TableHeader.js

https://github.com/ashvin27/react-datatable/blob/9768f04a84d88ddb31d4dd2a9f7515e598ead7fc/src/components/TableHeader.js#L20

**Comment by aiveneh** [comment]
this commit fixed it for me in my fork
https://github.com/aiveneh/react-datatable/commit/2241fafad89b6d98f7dcf54ef6f3c4d0ef7dfb5b

**Comment by ashvin27** [comment]
also fixed in version: 1.5.3
